screenpipe-app-tauri/lib/hooks/use-settings.tsx

```diff
@@ -140,213 +140,218 @@ export function useSettings() {
     }
   };
 
-  useEffect(() => {
-    const loadSettings = async () => {
-      if (!store) {
-        await initStore();
-      }
+  const resetSettings = async () => {
```

Comment:
@oliverqx you're just changing settings to default values of settings but some values are meant to be set dynamically as an example the `fps` value in `defaultSettings` object is 0.5 and this `resetSettings` function is going to set the `fps` value `0.5` on all os but it should change the values of fps dynamically depending upon os

```typescript
        const savedFps =
          (await store!.get<number>("fps")) ||
          (currentPlatform === "macos" ? 0.2 : 1);
```

also there are some unused imports!